README.md

```diff
@@ -20,7 +20,7 @@ To check the version, enter `kubectl version`.
 
 Install Kyverno:
 ```console
-kubectl create -f https://github.com/nirmata/kyverno/raw/master/definitions/install.yaml
+kubectl create -f https://raw.githubusercontent.com/nirmata/kyverno/v1.1.8/definitions/install.yaml
```

Comment:
Are we facing any issue here? The image tag used in master is the latest released version. If we change to this new URL, we'll need to update this doc for every new release.

Reply:
Can we use 
```
kubectl apply -f https://github.com/nirmata/kyverno/releases/download/<VESION>/install.yaml
```

Reply:
@realshuting - yes, the issue is that the current installation fails as the YAMLs are not compatible with the 1.1.8 version:

```yaml
kubectl create -f https://github.com/nirmata/kyverno/raw/master/definitions/install.yaml
namespace/kyverno created
customresourcedefinition.apiextensions.k8s.io/clusterpolicies.kyverno.io created
customresourcedefinition.apiextensions.k8s.io/clusterpolicyviolations.kyverno.io created
customresourcedefinition.apiextensions.k8s.io/generaterequests.kyverno.io created
customresourcedefinition.apiextensions.k8s.io/policyviolations.kyverno.io created
serviceaccount/kyverno-service-account created
clusterrole.rbac.authorization.k8s.io/kyverno:customresources created
clusterrole.rbac.authorization.k8s.io/kyverno:generatecontroller created
clusterrole.rbac.authorization.k8s.io/kyverno:policycontroller created
clusterrole.rbac.authorization.k8s.io/kyverno:userinfo created
clusterrole.rbac.authorization.k8s.io/kyverno:webhook created
clusterrole.rbac.authorization.k8s.io/kyverno:policyviolations created
clusterrole.rbac.authorization.k8s.io/kyverno:view-clusterpolicyviolations created
clusterrole.rbac.authorization.k8s.io/kyverno:view-policyviolations created
clusterrolebinding.rbac.authorization.k8s.io/kyverno:customresources created
clusterrolebinding.rbac.authorization.k8s.io/kyverno:generatecontroller created
clusterrolebinding.rbac.authorization.k8s.io/kyverno:policycontroller created
clusterrolebinding.rbac.authorization.k8s.io/kyverno:userinfo created
clusterrolebinding.rbac.authorization.k8s.io/kyverno:webhook created
configmap/init-config created
service/kyverno-svc created
deployment.apps/kyverno created
C:\Users\jbugwadia\Code\nirmata\java-model (bugfix/optimize_paginated_queries -> origin)
λ kubectl -n kyverno get pods
NAME                       READY   STATUS             RESTARTS   AGE
kyverno-7d5c86b76b-fdtnf   0/1     CrashLoopBackOff   1          10s
C:\Users\jbugwadia\Code\nirmata\java-model (bugfix/optimize_paginated_queries -> origin)
λ kubectl -n kyverno logs -f kyverno-7d5c86b76b-fdtnf
flag provided but not defined: -excludeGroupRole
Usage of /kyverno:
  -add_dir_header
        If true, adds the file directory to the header
  -alsologtostderr
        log to standard error as well as files
  -filterK8Resources string
        k8 resource in format [kind,namespace,name] where policy is not evaluated by the admission webhook. example --filterKind "[Deployment, kyverno, kyverno]" --filterKind "[Deployment, kyverno, kyverno],[Events, *, *]"
  -fqdn-as-cn
        use FQDN as Common Name in CSR
  -kubeconfig string
        Path to a kubeconfig. Only required if out-of-cluster.
  -log_backtrace_at value
        when logging hits line file:N, emit a stack trace
  -log_dir string
        If non-empty, write log files in this directory
  -log_file string
        If non-empty, use this log file
  -log_file_max_size uint
        Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
  -logtostderr
        log to standard error instead of files (default true)
  -profile
        Set this flag to 'true', to enable profiling.
  -runValidationInMutatingWebhook string
        Validation will also be done using the mutation webhook, set to 'true' to enable. Older kubernetes versions do not work properly when a validation webhook is registered.
  -serverIP string
        IP address where Kyverno controller runs. Only required if out-of-cluster.
  -skip_headers
        If true, avoid header prefixes in the log messages
  -skip_log_headers
        If true, avoid headers when opening log files
  -stderrthreshold value
        logs at or above this threshold go to stderr (default 2)
  -v value
        number for the log level verbosity
  -vmodule value
        comma-separated list of pattern=N settings for file-filtered logging
  -webhooktimeout int
        timeout for webhook configurations (default 3)
C:\Users\jbugwadia\Code\nirmata\java-model (bugfix/optimize_paginated_queries -> origin)
```

So we either need to change the image tag to `latest` or allow users to install a specific version of a the YAMLs.

Reply:
Ok, I see. 
Similar to @evalsocket suggestion, we can set the ENV "version" to the latest tag and install from there:
````
export version=$(curl --silent "https://github.com/nirmata/kyverno/$1/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')

kubectl create -f https://raw.githubusercontent.com/nirmata/kyverno/$version/definitions/install.yaml
````

Reply:
@realshuting changes are good to go if we use release url 
```
export version=$(curl --silent "https://github.com/nirmata/kyverno/$1/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')
kubectl create -f https://github.com/nirmata/kyverno/releases/download/$version/install.yaml
```

Reply:
I have some suggestions to use `latest` tag in install.yaml and in release process use kustomize to add version tag before uploading them to release

Reply:
My concern with latest tag is that the image might be updated if the pod somehow gets restarted on the user side. This will cause the same issue as we've seen.